src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -3058,6 +3058,9 @@ bool AdaptDispatch::WindowManipulation(const DispatchTypes::WindowManipulationTy
     case DispatchTypes::WindowManipulationType::ResizeWindowInCharacters:
         _api.ResizeWindow(parameter2.value_or(0), parameter1.value_or(0));
         return true;
+    case DispatchTypes::WindowManipulationType::ReportTextSizeInCharacters:
+        _api.ReturnResponse(fmt::format(L"\033[8;{};{}t", _api.GetViewport().height(), _api.GetTextBuffer().GetSize().Width()));
```

Comment:
nit: all of the other uses of `fmt` in this file use `FMT_COMPILE` for the format string. That keeps the code size down when this is compiled into Windows' conhost.exe.

Would you mind doing the same? :smile: